examples/social/web-frontend/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with the Social App.
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+The application (contract/service) must be deployed on the network, and a Node Service must be running
+on port 8000. Also make sure you have performed the operations mentoined in this [README](../README.md)
+
+### Note
+
+The app is an example of a social app, and will be improved with future updates. Users can see their post by `ownPost` query in GraphiQL IDE.
+
+#### Operations avaiable:
+
+Post, Likes, and Comments (only by the creation chain).
+
+To access the social app for a given Application Id, append the Application Id and port to the URI as so:
+
+```
+http://localhost:3000/7817752ff06b8266d77df8febf5c4b524cec096bd83dc54f989074fb94f833737ae984f32be2cee1dfab766fe2d0c726503c4d97117eb59023e9cc65a8ecd1f7000000000000000000000000000000007817752ff06b8266d77df8febf5c4b524cec096bd83dc54f989074fb94f833737ae984f32be2cee1dfab766fe2d0c726503c4d97117eb59023e9cc65a8ecd1f702000000000000000000000000000000&port=8000
```

Comment:
This is not quite correct, the format is `https://localhost:8080/{chain_id}?app={app ID}&owner={owner ID}&port={node service port}`.  So running with the default settings from `examples/social/README.md` I had to navigate to:

```
http://localhost:8080/e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65?app=e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65010000000000000001000000e476187f6ddfeb9d588c7b45d3df334d5501d6499b3f9ad5595cae86cce16a65030000000000000000000000&owner=47f3d51e4315d75ceb5eafd05c63688cd150740adb8f26dcb8de3d54af07b992
```

Reply:
:( i have updated the `README` file.

Reply:
Also you can create a post with `imageUrl`, it looks good :)